inc/richtext/richtext.class.php

```diff
@@ -234,4 +235,26 @@ private static function normalizeHtmlContent(string $content, bool $enhanced_htm
 
       return $content;
    }
+
+   public static function getEnhancedHtml(string $content, array $params = []): string {
+      $p = [
+         'images_gallery'           => false,
+         'user_mentions'            => true,
+         'encode_output_entities'   => false,
```

Comment:
Enhanced HTML is made to be displayed in a rendering context, so encoding entities in output should never be used. You could remove this option, to prevent a misusage of it.
```suggestion
```